sdks/python/setup.py

```diff
@@ -119,7 +119,9 @@ def get_version():
     'oauth2client>=2.0.1,<4',
     'protobuf>=3.5.0.post1,<4',
     # [BEAM-6287] pyarrow is not supported on Windows for Python 2
-    ('pyarrow>=0.11.1,<0.15.0; python_version >= "3.0" or '
+    # [BEAM-8368] pyarrow>=0.14.0 triggers an exception when importing
+    # apache_beam on macOS 10.15
```

Comment:
Would you mind filing a jira with arrow and referencing it here? It looks like arrow is [considering](https://lists.apache.org/thread.html/462464941d79e67bfef7d73fc900ea5fe0200e8a926253c6eb0285aa@%3Cdev.arrow.apache.org%3E) an 0.15.1 release so maybe this could be resolved there.

Reply:
Done, jira is filled.

Reply:
https://issues.apache.org/jira/browse/BEAM-8392